src/components/CartIcon/index.js

```diff
@@ -0,0 +1,11 @@
+import { connect } from 'react-redux';
+
+import CartIcon from './CartIcon';
+
+function mapStateToProps(state) {
```

Comment:
`function mapStateToProps({ cart })` fot shorter assignment :D

Reply:
Not a big fan of destructuring here.